.travis.yml

```diff
@@ -6,8 +6,10 @@ language: cpp
 matrix:
     include:
         - os: osx
+          env: PLATFORM=osx
```

Comment:
Right now osx platform will build for osx and ios, should we add another platform to separate osx and ios?